gremlin-python/src/main/python/tests/driver/test_driver_remote_connection.py

```diff
@@ -40,8 +40,7 @@ class TestDriverRemoteConnection(object):
     # this is a temporary test for basic graphSONV4 connectivity, once all types are implemented, enable graphSON testing
     # in conftest.py and remove this
     def test_graphSONV4_temp(self):
-        remote_conn = DriverRemoteConnection(test_no_auth_url, 'gmodern',
-                                             message_serializer=serializer.GraphSONSerializerV4())
+        remote_conn = DriverRemoteConnection(test_no_auth_url, 'gmodern')
```

Comment:
Does removing the serializer param change the test serializer from graphSON to graph binary since graph binary is the default?

Reply:
Thanks for catching this, I'll update this to set both the request and response serializers to GraphSON.

Reply:
I wonder why the test didn't fail when the serializer was changed to graph binary?

Reply:
Those queries work for either graphson or graphbinary so there wouldn't be a failure, just a gap in testing.

Reply:
Makes sense and I don't see an easy way for the test to verify that the given serializer was used.